resources/fuel_library/actions/run.sh

```diff
@@ -11,10 +13,15 @@ else
     git pull
     popd
 fi
+
+[ -n $use_librarian_puppet_simple ] && gem install librarian-puppet-simple --no-ri --no-rdoc
+
 pushd ./fuel-library/deployment
 ./update_modules.sh
 popd
 
+[ -n $use_librarian_puppet_simple ] && gem uninstall -x librarian-puppet-simple
```

Comment:
this package actually replaces librarian_puppet, so I install it and then remove w/o any harm to librarian-puppet